### PR TITLE
Add new OXM: IFP Class ID.

### PR DESCRIPTION
**Pull request by wilmo119**
Reviewer: @shudongz @andi-bigswitch @rizard

### CODE REVIEW

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-566-jenkins-loxigen_pull_req_artifacts-118

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/118/

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/135/

File: openflow_input/bsn_ifp_class_id [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS
+// IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the EPL for the specific language
+// governing permissions and limitations under the EPL.
+
+#version 3
+#version 4
+#version 5
+#version 6
```
Comment (shudongz) [inline openflow_input/bsn_ifp_class_id]: Should we do it just for version 5 & 6?

**Comment by shudongz** [comment]
LGTM, only the version question.

**Comment by wilmo119** [comment]
SELF-ACK

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-566-jenkins-loxigen_pull_req_artifacts-119

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/119/

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/136/

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2017.09.18.2244-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2017.09.18.2244-m.loxigen.master/abat.log
/cc

**Pull request closed without merging**